coordinator/hardhat.config.js

```diff
@@ -7,12 +7,19 @@ const path = require("path");
 dotenv.config();
 
 const parentDir = __dirname.includes("build") ? ".." : "";
+const TEST_MNEMONIC = "test test test test test test test test test test test junk";
```

Comment:
that's also how I imagine it works, but it's the only way I can let it work, not really sure why

Reply:
this is because of the need for a signer object https://github.com/privacy-scaling-explorations/maci/blob/dev/coordinator/ts/proof/proof.service.ts#L64 - the mnemonic can be from any account without $, it's just needed for the signer object

Reply:
Check how it's done for ci job or for docker container. It works there without any mnemonics.

Reply:
that works because you are running the hardhat node with this env variable RPC url set to localhost https://github.com/privacy-scaling-explorations/maci/blob/dev/coordinator/.env.example#L25
if you change that to an actual RPC url for a testnet/mainnet then it won't work